src/main/java/teammates/ui/controller/InstructorCourseEditPageData.java

```diff
@@ -7,23 +8,119 @@
+                                                                  Const.Tooltips.COURSE_INSTRUCTOR_REMIND, 
+                                                                  "return toggleSendRegistrationKey('" + instructor.courseId + "','" + instructor.email + ");",
+                                                                  isDisabled);
+            }
+            
+            buttonContent = "<span class=\"glyphicon glyphicon-pencil\"></span> Edit";
+            isDisabled = !this.currentInstructor.isAllowedForPrivilege(Const.ParamsNames.INSTRUCTOR_PERMISSION_MODIFY_INSTRUCTOR);
+            instructorPanel.editButton = createButton(buttonContent, "btn btn-primary btn-xs",
+                                                      "instrEditLink" + instructorPanel.index, "javascript:;", 
+                                                      Const.Tooltips.COURSE_INSTRUCTOR_EDIT,
+                                                      null, isDisabled);
+            
+            buttonContent = "<span class=\"glyphicon glyphicon-trash\"></span> Delete";
+            isDisabled = !this.currentInstructor.isAllowedForPrivilege(Const.ParamsNames.INSTRUCTOR_PERMISSION_MODIFY_INSTRUCTOR);
+            instructorPanel.deleteButton = createButton(buttonContent, "btn btn-primary btn-xs", 
+                                                        "instrDeleteLink" + instructorPanel.index, 
+                                                        this.getInstructorCourseInstructorDeleteLink(instructor.courseId, instructor.email), 
+                                                        Const.Tooltips.COURSE_INSTRUCTOR_DELETE, 
+                                                        "return toggleDeleteInstructorConfirmation('" + instructor.courseId + "','" + instructor.email + "', " + instructor.email.equals(this.account.email) + ");",
+                                                        isDisabled);
+        }
+        
+        return instructorPanel;
+    }
+    
+    public ElementTag getDeleteCourseButton() {
+        return deleteCourseButton;
+    }
+    
+    public ElementTag getAddInstructorButton() {
+        return addInstructorButton;
+    }
+    
+    public CourseEditInstructorPanel getAddInstructorPanel() {
+        return addInstructorPanel;
+    }
+    
+    public CourseAttributes getCourse() {
+        return course;
+    }
+    
+    public List<CourseEditInstructorPanel> getInstructorPanelList() {
+        return instructorPanelList;
+    }
+    
+    public List<InstructorAttributes> getInstructorList() {
+        ArrayList<InstructorAttributes> instructorList = new ArrayList<InstructorAttributes>();
+        for (CourseEditInstructorPanel instructorPanel : instructorPanelList) {
+            instructorList.add(instructorPanel.instructor);
+        }
+        return instructorList;
+    }
```

Comment:
Is this method used anywhere except in the tests?
If the tests require the data in a certain format, it should convert the data itself (i.e. call `getInstructorPanelList()` and compare each of the `instructor` strings from them).